hooks/pre/30-upgrade.rb

```diff
@@ -212,6 +240,7 @@ def fail_and_exit(message)
   if katello || foreman_proxy_content
     upgrade_step :upgrade_qpid_paths
     upgrade_step :migrate_pulp, :run_always => true
+    upgrade_step :mongo_mmapv1_check
```

Comment:
From my testing, this needs to be the last step we take, I moved it to line 258.